src/lib/tooltip/tooltip.scss

```diff
@@ -2,23 +2,23 @@
 @import '../core/a11y/a11y';
 
 
-$md-tooltip-height: 22px;
+$md-tooltip-target-height: 22px;
+$md-tooltip-font-size: 10px;
 $md-tooltip-margin: 14px;
-$md-tooltip-padding: 8px;
+$md-tooltip-horizontal-padding: 8px;
+$md-tooltip-vertical-padding: ($md-tooltip-target-height - $md-tooltip-font-size) / 2;
 
 :host {
   pointer-events: none;
 }
 
 .md-tooltip {
-  color: white;
-  padding: 0 $md-tooltip-padding;
+  color: #fff;
```

Comment:
I prefer using `white` since it's more meaningful; the css minifier will condense it.